servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/BufferStrategiesTest.java

```diff
@@ -80,7 +80,7 @@
     static final ExecutorExtension<Executor> EXECUTOR_EXTENSION = withCachedExecutor().setClassLevel(true);
 
     @RegisterExtension
-    final ExecutorExtension<TestExecutor> testExecutorExtension = withTestExecutor();
+    static final ExecutorExtension<TestExecutor> testExecutorExtension = withTestExecutor();
```

Comment:
If we can not use `.setClassLevel(true)` in this case or any other, it's better to keep it as non-static variable. Otherwise, it's harder to understand the test flow and when this extension is re-initialized.